velox/serializers/PrestoSerializer.cpp

```diff
@@ -1720,11 +1800,22 @@ std::unique_ptr<VectorSerializer> PrestoVectorSerde::createSerializer(
     int32_t numRows,
     StreamArena* streamArena,
     const Options* options) {
```

Comment:
Does it require a lot of changes if we make options a const &? and provide a default one on ctor if not specified?

Reply:
This will be a lot of change and non-relevant change, we can do that in another PR if needed.